core/src/actors/storage_manager.rs

```diff
@@ -13,6 +13,9 @@ use witnet_util::error::WitnetError;
 type ValueStorageResult<T> = StorageResult<Option<T>>;
 type UnitStorageResult = StorageResult<()>;
 
+/// Constant to specify the peers key for the storage
+pub static PEERS_KEY: &'static [u8] = b"peers";
```

Comment:
I don't think this constant belongs here. We may need a separate file somewhere for static storage keys.